specs/agents/tracing-instrumentation-db.md

```diff
@@ -188,6 +188,23 @@ If they don't, they MUST use a hard-coded list of URLs that correspond with the
 |`_.type`| `mongodb` | |
 |`_.name`| e.g. `customers` | Database name, same as `db.instance` if available  |
 
+#### `mongodb_capture_statement_commands` configuration
+
+Agents that support capturing MongoDB statements SHOULD implement this option.
+
+Statement capture SHOULD be disabled by default to avoid performance overhead and capturing potentially sensitive data.
+
```

Comment:
Seems to be outdated as the new default is to capture certain commands by default.
```suggestion
```